integrations/spark/openhouse-spark-runtime/build.gradle

```diff
@@ -45,6 +45,14 @@ dependencies {
     exclude group: 'org.roaringbitmap'
   }
 
+  testImplementation("org.apache.iceberg:iceberg-spark-runtime-3.1_2.12:" + icebergVersion)
+  testImplementation(project(':tables-test-fixtures_2.12'))
```

Comment:
Curious, why tables-test-fixtures need to be included here?

Reply:
becoz `integrations/spark/openhouse-spark-runtime/src/test/java/CatalogOperationTest.java` is moved into this module